ui-v2/package.json

```diff
@@ -3,24 +3,25 @@
   "version": "2.2.0",
   "private": true,
   "description": "The web UI for Consul, by HashiCorp.",
+  "repository": "",
   "directories": {
     "doc": "doc",
     "test": "tests"
   },
-  "repository": "",
   "scripts": {
     "build": "ember build --environment production",
     "build:staging": "ember build --environment staging",
-    "build-ci": "ember build --environment test",
+    "build:ci": "ember build --environment test",
```

Comment:
the `GNUMakefile` still uses `yarn run build-ci`, do you need to update that too?

Reply:
Oh wow, such a good spot @gregone thanks! I've amended in a commit on the end (I just changed it the one that I left to `:`. Now I can see why it was duplicated in the cherrypick!